ui/app/templates/permissions_modal.html

```diff
@@ -58,7 +58,7 @@ <h3 class="panel-title">
         <form role="form" ng-submit="grantRole()">
           <div class="form-group" ng-class="{'has-error': errors.role}">
             <label for="id_role">Role</label>
-            <input type="text" class="form-control" id="id_role" ng-model="role.role"
```

Comment:
I think https://bugzilla.mozilla.org/show_bug.cgi?id=1341622 covers part of what you're talking about here.

I do agree that it's confusing to have the top level nav be called "Permissions", but the content talk about Users. Perhaps we should change the nav button, now that this section covers more than just Permissions?